### DISCUSSION TITLE
Sonoff v1.1 ZbBridge Module restarts every 16s

### DISCUSSION THREAD

**Discussion by HaroldasJas** [arendst/Tasmota#11177] [Q&A] [answered]
Program Version | 9.3.1.1(zbbridge) - the same problem with 9.3.1.  (9.2.0 - works stable)
Build Date & Time | 2021-02-28T22:04:01
Core/SDK Version | 2_7_4_9/2.2.2-dev(38a443e)

If module parameters are not Sonoff ZbBridge (75), hardware works stable.

Log:
00:00:01.756 RSL: INFO1 = {"Module":"Sonoff ZbBridge","Version":"9.3.1(zbbridge)","FallbackTopic":"cmnd/DVES_D7F36B_fb/","GroupTopic":"cmnd/tasmotas/"}
00:00:01.758 RSL: INFO2 = {"WebServerMode":"Admin","Hostname":"SonoffZBB","IPAddress":"192.168.1.118"}
00:00:01.761 RSL: INFO3 = {"RestartReason":{"Exception":9,"Reason":"Exception","EPC":["40238f4b","00000000","00000000"],"EXCVADDR":"00004bca","DEPC":"00000000","CallChain":["40238dcc","4020e85c","4020e855","402448f6","40103587","40103764","402448f6","40103c1e","402440d8","40100828","4024414c","40238d72","401004a4","40102a00","40105c93","401004a4","40103587","4010652d","40103764","40103c1e","401004a4","40102a00","40105c93","40103587","401004a4","40103764","401004a4","40103c1e","401008c0","4027006e","4020ba44"]}}
14:15:13.339 QPC: Reset
14:15:13.343 RSL: STATE = {"Time":"2021-03-01T14:15:13","Uptime":"0T00:00:08","UptimeSec":8,"Vcc":3.510,"Heap":30,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":0,"Wifi":{"AP":1,"SSId":"Speed5G","BSSId":"30:23:03:97:5D:FB","Channel":4,"RSSI":46,"Signal":-77,"LinkCount":1,"Downtime":"0T00:00:03"}}

**Comment by ascillato2** [comment, marked as answer]
Hi,

This is an issue more than a question or an idea to discuss. Can you please open a new issue completing the [troubleshooting template](https://github.com/arendst/Tasmota/issues/new/choose) in order to have more information so as to properly help you?

Remember to read the [Contributing Guideline and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md). Thanks.

---------------------------

### Support Information 
See [Docs](https://tasmota.github.io/docs/) for more information.
See [Chat](https://discord.gg/Ks2Kzd4) for more user experience.
See [Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
